services/user.py

```diff
@@ -0,0 +1,55 @@
+from django.db.models import QuerySet
+from db.models import User
+from django.contrib.auth import get_user_model
+
+
+def create_user(
+    username: str,
+    password: str,
+    email: str = None,
+    first_name: str = None,
+    last_name: str = None,
+) -> None:
+    user = get_user_model().objects.create_user(username=username)
+    if password:
+        user.set_password(password)
+
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+    user.save()
+
+
+def get_user(user_id: int) -> QuerySet:

```

Comment:
function returns User `-> User`
Remember:
`.all()`, `.filter()` returns QuerySet
`.get()`, `.delete()`, `.create()` returns model class, `-> User` for. example